Makefile

```
@@ -1,8 +1,18 @@
 IMAGE_NAME=quay.io/theforeman/candlepin
 IMAGE_TAG=4.4.14
 
+PROJECT_XY_TAG=4.4
+PROJECT_XYZ_TAG=4.4.14
```

I'm wondering if it makes sense to avoid the duplication
```suggestion
PROJECT_XYZ_TAG=${PROJECT_XY_TAG}.14
```